Mathlib/Algebra/Order/Support.lean

```diff
@@ -241,4 +242,18 @@ lemma abs_indicator_symmDiff (s t : Set α) (f : α → M) (x : α) :
   apply_indicator_symmDiff abs_neg s t f x
 
 end LinearOrderedAddCommGroup
+
+section NormedAddCommGroup
+variable [NormedAddCommGroup M]
+
+open scoped symmDiff
+
+/-- The norm of an indicator of the symmetric difference is equal to the norm of the difference of
+the two indicators. -/
+lemma norm_indicator_symmDiff (s t : Set α) {f : α → M} :
```

Comment:
Well indeed it is a direct application, but I thought it would be useful to have it directly. I prove equality of functions, so if I am doing some rewrites with functions which are not evaluated I don't have to stop the `rw`. In my proof of `norm_indicatorConstLp_sub`, if I do not use `norm_indicator_symmDiff`, I have to do something like this:
```lean
import Mathlib.MeasureTheory.Function.LpSpace

open ENNReal symmDiff MeasureTheory

theorem norm_indicatorConstLp_sub {α E : Type*} [MeasurableSpace α] {μ : Measure α}
    {s t : Set α} (hs : MeasurableSet s) (ht : MeasurableSet t) (hμs : μ s ≠ ∞) (hμt : μ t ≠ ∞)
    [NormedAddCommGroup E] (c : E)
    (hp_ne_zero : p ≠ 0) (hp_ne_top : p ≠ ∞) :
    ‖indicatorConstLp p hs hμs c - indicatorConstLp p ht hμt c‖ =
    ‖c‖ * (μ (s ∆ t)).toReal ^ (1 / p.toReal) := by
  rw [indicatorConstLp, indicatorConstLp, ← Memℒp.toLp_sub, Lp.norm_toLp,
    ← snorm_norm]
  simp_rw [Pi.sub_apply, ← Set.apply_indicator_symmDiff norm_neg]
  rw [snorm_norm, snorm_indicator_const (hs.symmDiff ht) hp_ne_zero hp_ne_top, ENNReal.toReal_mul,
    toReal_coe_nnnorm, ← ENNReal.toReal_rpow]
```
So I stop the `rw` to make a `simp_rw` instead.
Also `norm_indicator_symmDiff` is very similar to `abs_indicator_symmDiff` just above, which is the same with absolute value, so I thought it would make sense to have it with the norm as well.
Tell me if you still think I should remove that lemma.

Reply:
On second thought, I see what you mean. I do not have a lot of experience and I do not like to stop a `rw` a sequence because some lemma is not exactly in the right form, but I cannot add every lemma to have exactly the right form.